src/CombinedRes.cpp

```diff
@@ -69,16 +69,39 @@ std::complex<long double> CombinedRes::Matching(std::complex<long double> N,
 
 std::complex<long double> CombinedRes::CombinedResExpr(
     std::complex<long double> N, long double pt, int scheme) {
+  double pp = static_cast<double>(pt);
+  // take only real part. Does not work for complex
+  double nn = static_cast<double>(N.real());
   std::complex<long double> mres;
+  std::vector<double> ResultsMellin;
+  std::vector<double> zero(2, 0.0);
+
```

Comment:
What does setting SigmaLO from hpt-mon to 2 mean here?

Reply:
2 is the length of the vector

Reply:
OMG, I was being stupid here. I wrote it myself and forgot what it meant :rofl: I thought it was a complex number.